Documentation/install/rhel/installing-workers.md

```diff
@@ -91,15 +91,16 @@ After the `tectonic-release` RPM is installed, complete the installation of the
 $ yum install tectonic-worker
 ```
 
-This will download the relevant dependencies and then prompt to validate the
-GPG key installed by the `tectonic-release` RPM.
+This will download the relevant dependencies and then prompt to validate the GPG key installed by the `tectonic-release` RPM.
 
-The Kubelet version must be kept in sync with the cluster's Tectonic version. To prevent incorrect `yum update` commands from updating the version, disable the Tectonic repo by writing `enabled=0` into:
+The kubelet version should be kept in sync with the cluster's Tectonic version. To prevent `yum upgrade` commands from unexpectedly updating the version, the Tectonic repo can be disabled by setting `enabled=0` in `/etc/yum.repos.d/tectonic.repo`. The Tectonic packages can then be upgraded by running:
```

Comment:
I wanted to call out `/etc/yum.repos.d/tectonic.repo` on a separate line in case people are skimming stuff. I worry that this is important and might be missed when in the paragraph. Maybe even list out a command to set this value as a oneliner

Reply:
I think I am going to rewrite this into a new section that also explains scenarios like fixing versions for specific Tectonic channels, e.g. to continue getting updates but only for Tectonic 1.6.